moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -157,346 +134,276 @@ export const useColumns = ({ hiddenColumns }) => {
-              {contractor}
-            </span>
-          ));
-        },
+        renderCell: ({ row }) =>
+          renderSplitListDisplayBlock(row, "workgroup_contractors"),
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.medium,
       },
       {
-        title: "Contract numbers",
+        headerName: "Contract numbers",
         field: "contract_numbers",
-        hidden: hiddenColumns["contract_numbers"],
-        emptyValue: "-",
-        cellStyle: { whiteSpace: "noWrap" },
-        render: (entry) => {
-          return entry.contract_numbers.split(",").map((contractNumber, i) => (
-            <span key={i} style={{ display: "block" }}>
-              {contractNumber}
-            </span>
-          ));
-        },
+        renderCell: ({ row }) =>
+          renderSplitListDisplayBlock(row, "contract_numbers"),
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.medium,
       },
       {
-        title: "Tags",
+        headerName: "Tags",
         field: "project_tags",
-        hidden: hiddenColumns["project_tags"],
-        cellStyle: { whiteSpace: "noWrap" },
-        emptyValue: "-",
-        render: (entry) => {
-          return entry.project_tags.split(",").map((tag) => (
-            <span key={tag} style={{ display: "block" }}>
-              {tag}
-            </span>
-          ));
-        },
+        renderCell: ({ row }) =>
+          renderSplitListDisplayBlock(row, "project_tags"),
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.large,
       },
       {
-        title: "Created by",
+        headerName: "Created by",
         field: "added_by",
-        hidden: hiddenColumns["added_by"],
-        emptyValue: "-",
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.small,
       },
       {
-        title: "Public process status",
+        headerName: "Public process status",
         field: "public_process_status",
-        hidden: hiddenColumns["public_process_status"],
-        emptyValue: "-",
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.small,
       },
       {
-        title: "Interim MPD (Access) ID",
+        headerName: "Interim MPD (Access) ID",
         field: "interim_project_id",
-        hidden: hiddenColumns["interim_project_id"],
-        emptyValue: "-",
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.small,
       },
       {
-        title: "Components",
+        headerName: "Components",
         field: "components",
-        hidden: hiddenColumns["components"],
-        emptyValue: "-",
-        render: (entry) => filterComponentFullNames(entry),
+        renderCell: ({ row }) => filterComponentFullNames(row),
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.medium,
       },
       {
-        title: "Parent project",
-        field: "parent_project_id",
-        hidden: hiddenColumns["parent_project_id"],
-        emptyValue: "-",
-        render: (entry) => (
+        headerName: "Parent project",
+        field: "parent_project_name",
+        renderCell: ({ row }) => (
```

Comment:
sorry to make this change as you were reviewing this, but I swapped the field from the parent_project_id to the parent_project_name so the sorting would work. previously it was sorting on the parent project id and displaying the name. now it sorts on the name and displays the name.

Reply:
all good! i can take a closer look tomorrow, but I'm not sure if the sort is acting quite like we expect it too. i think that I'm seeing a difference between the order of the returned results and what is displayed.

Reply:
I reverted back to its old ways, and turned off the sorting. This can be a refinement later, I wanna get this in to moped.